cl/sentinel/gossip.go

```diff
@@ -123,7 +152,14 @@ func (s *GossipManager) Recv() <-chan *GossipMessage {
 func (s *GossipManager) GetMatchingSubscription(match string) *GossipSubscription {
 	var sub *GossipSubscription
 	s.subscriptions.Range(func(topic, value interface{}) bool {
-		if strings.Contains(topic.(string), match) {
+		topicStr := topic.(string)
+		// take out third part of the topic by splitting on "/"
+		// reference: eth2/d31f6191/beacon_attestation_45/ssz_snappy
```

Comment:
seems the example is wrong becasue miss the leading '/'
`/eth2/d31f6191/beacon_attestation_45/ssz_snappy`